symbolicationWebService.py

```diff
@@ -125,42 +159,24 @@ def do_POST(self):
         return
 
       LogDebug("Request body: " + requestBody)
-      rawRequest = json.loads(requestBody)
 
-      request = SymbolicationRequest(gSymFileManager, rawRequest)
-      if not request.isValidRequest:
+      response = yield gPool.submit(processSymbolicationRequest, requestBody)
+      if response is None:
         LogDebug("Unable to parse request")
```

Comment:
how hard would it be to print the realIP for every debug message generated while handling a request?